#Custom message for our sensors and controls node.

#DVL Data
float32 DVL_pitch
float32 DVL_yaw
float32 DVL_roll
float32 DVL_x_translation
float32 DVL_y_translation

#IMU data
float32 IMU_pitch
float32 IMU_yaw
float32 IMU_roll

#Barometer data
float32 barometer_depth
float32 barometer_temperature

#Sonar data
float32 sonar_distance
float32 sonar_confidence


#Hydrophones data
float32 hydrophones_direction

#Thruster data
float32 thruster_horisontal_front_left
float32 thruster_horisontal_front_right
float32 thruster_horisontal_back_left
float32 thruster_horisontal_back_right
float32 thruster_vertical_front_left
float32 thruster_vertical_front_right
float32 thruster_vertical_back_left
float32 thruster_vertical_back_right
